src/UI/COSetupController.java

```diff
@@ -94,6 +95,20 @@ public boolean handleInput(InputAction action)
             coSelectors[playerSelector.getSelectionNormalized()].setSelectedOption(0);
         }
         break;
+      case SEEK: // Display the CO Info menu so we can window-shop
```

Comment:
From the CO Setup screen, I kinda want the info screen to cycle through all available COs, not just those that are already selected... at least, that's what I naturally expected when I tried it. It feels a bit cumbersome to back out to cycle to a different person. Maybe it's just me.

I kinda want to rework this whole screen anyway if I ever find time, so don't worry too much about messing with it right now.

Reply:
Hmm. That would make sense. Just import the entire list of COs, and start from the currently-selected one.
Having it at all is better than the carts, though, haha.

> I kinda want to rework this whole screen anyway if I ever find time, so don't worry too much about messing with it right now.

I'm glad you think so highly of my designs :P

Reply:
Mostly I'd like to be a bit more space efficient, remove the map, and include more info (such as actually showing the infantry sprite being selected). Probably also make a better way to select COs/Colors/Sprites than viewing one at a time when the list is potentially huge.

Reply:
> remove the map

Hmm. It does take a lot of space, but IMO it'll be useful with arbitrary players in unknown positions: you could change the color to see which player is where on the map (obviously that'd require that be done, though).

Reply:
That's a fair point, but then AW doesn't show the map during CO select, and I don't think I've ever felt the need for it there (the exceptions being those final missions where you pick three COs to gang up on the big bad, but those are different).
Maybe I'll send you mockups of what I have in mind. We can keep the map if you think there's value, but I might still want to tweak some things.